src/layer/S2L2ALayer.ts

```diff
@@ -37,4 +50,20 @@ export class S2L2ALayer extends AbstractSentinelHubV3WithCCLayer {
         .join(''),
     };
   }
+
+  protected extractFindTilesMetaFromCatalog(feature: Record<string, any>): Record<string, any> {
+    let result: Record<string, any> = {};
+
+    result = {
+      ...super.extractFindTilesMetaFromCatalog(feature),
+    };
+
+    if (feature.assets && feature.assets.data) {
```

Comment:
With S-2 L2A you also check `&& feature.assets.data.href {`... Not sure if it is needed, just wanted to point out inconsistency if it is a bug. Fftr.

Reply:
Missing check was added